src/controllers/orders.js

```diff
@@ -0,0 +1,6 @@
+const Order = require('../models/order');
+
+exports.getMyOrders = async (req, res, next) => {
+  const myOrders = await Order.find({ buyer: req.user._id });
+  return res.status(200).json({ success: true, data: myOrders });
```

Comment:
If you try to execute this endpoint using postman, you'd see that `orderItems ` are objectIds. 
The entity/frontend that consumes the API, wouldn't be interested in objectIds, because he doesn't know what they mean. So they should be populated.